components/dialogs/AuthDialog.vue

```diff
@@ -855,10 +856,18 @@ export default {
       await this.setAuthCoreToken(accessToken);
       this.currentTab = 'loading';
       this.platform = 'authcore';
+      let user = currentUser;
+      if (!user || !user.profileName) {
+        try {
+          user = await this.fetchAuthCoreUser();
+        } catch (err) {
+          console.error(err);
```

Comment:
continue the execution with no user?

Reply:
yes, the `user` is useful for frontend prefill purpose. As long as we have `id_token` and `access_token` we can handle authentication and email in backend.